File: 351002/Matsuyeu/Discussion/src/NewsPortal.Discussion.Application/Dtos/ErrorResponse.cs
namespace Discussion.src.NewsPortal.Discussion.Application.Dtos;

public class ErrorResponse
{
    public string ErrorMessage { get; set; } = string.Empty;
    public string ErrorCode { get; set; } = string.Empty;

    public ErrorResponse(string errorMessage, string errorCode)
    {
        ErrorMessage = errorMessage;
        ErrorCode = errorCode;
    }
}

File: 351004/Dashkevich/NoSQL_lab/discussion/src/main/java/com/adashkevich/nosql/lab/discussion/dto/ErrorResponse.java
package com.adashkevich.nosql.lab.discussion.dto;

public class ErrorResponse {
    public String errorMessage;
    public String errorCode;

    public ErrorResponse(String errorMessage, String errorCode) {
        this.errorMessage = errorMessage;
        this.errorCode = errorCode;
    }
}
